### ISSUE TITLE
New tab won't open

### ISSUE THREAD

**Issue by vulgrim** [kovidgoyal/kitty#520]
When I tried to open a new tab it prompted me for a new tab name, after entering the name  new tab did not open. 
I installed kitty for AUR on Arch linux. Kitty version installed is 0.9.0. 

Config file:

```# vim:fileencoding=utf-8:ft=conf

font_family      FuraCode Nerd Font Mono 
italic_font      auto
bold_font        auto
bold_italic_font auto

# Font size (in pts)
font_size        8.2

# The amount the font size is changed by (in pts) when increasing/decreasing
# the font size in a running terminal.
font_size_delta 2

adjust_line_height 0
adjust_column_width 0

cursor_shape    underline 
cursor_blink_interval     0

cursor_stop_blinking_after 15.0
scrollback_lines 2000
scrollback_pager less +G -R

wheel_scroll_multiplier 5.0
url_color #0087BD
url_style curly

# The modifier keys to press when clicking with the mouse on URLs to open the URL
open_url_modifiers kitty_mod

# The program with which to open URLs that are clicked on. The special value "default" means to
# use the operating system's default URL handler.
open_url_with default

copy_on_select no

# The modifiers to use rectangular selection (i.e. to select text in a
# rectangular block with the mouse)
rectangle_select_modifiers ctrl+alt

# Characters considered part of a word when double clicking. In addition to these characters
# any character that is marked as an alpha-numeric character in the unicode
# database will be matched.
select_by_word_characters :@-./_~?&=%+#

# The interval between successive clicks to detect double/triple clicks (in seconds)
click_interval 0.5

# Hide mouse cursor after the specified number of seconds of the mouse not being used. Set to
# zero to disable mouse cursor hiding.
mouse_hide_wait 3.0

focus_follows_mouse yes 

repaint_delay    10
input_delay 3
sync_to_monitor no 
visual_bell_duration 0.0
enable_audio_bell no 
remember_window_size   no 
initial_window_width   585
initial_window_height  300

window_border_width 0

# The window margin (in pts) (blank area outside the border)
window_margin_width 0

# The window padding (in pts) (blank area between the text and the window border)
window_padding_width 16

# The color for the border of the active window
active_border_color #00ff00

# The color for the border of inactive windows
inactive_border_color #cccccc

# Fade the text in inactive windows by the specified amount (a number between
# zero and one, with 0 being fully faded).
inactive_text_alpha 0.8
# }}}


# Tab bar {{{
# Which edge to show the tab bar on, top or bottom
tab_bar_edge bottom

# The separator between tabs in the tab bar
tab_separator " ┇"

# Tab bar colors and styles
active_tab_foreground #000
active_tab_background #eee
active_tab_font_style bold-italic
inactive_tab_foreground #444
inactive_tab_background #999
inactive_tab_font_style normal
# }}}


# Color scheme {{{
# The foreground color
# }}}

close_on_child_death no
allow_remote_control no
term xterm-kitty

kitty_mod ctrl+alt
map kitty_mod+v        paste_from_clipboard
map kitty_mod+s        paste_from_selection
map kitty_mod+c        copy_to_clipboard
map shift+insert       paste_from_selection

map kitty_mod+o        pass_selection_to_program


# Scrolling {{{
map kitty_mod+up        scroll_line_up
map kitty_mod+down      scroll_line_down
map kitty_mod+k         scroll_line_up
map kitty_mod+j         scroll_line_down
map kitty_mod+page_up   scroll_page_up
map kitty_mod+page_down scroll_page_down
map kitty_mod+home      scroll_home
map kitty_mod+end       scroll_end
map kitty_mod+h         show_scrollback
# }}}

# Window management {{{
map kitty_mod+enter    new_window
map kitty_mod+n        new_os_window
map kitty_mod+w        close_window
map kitty_mod+]        next_window
map kitty_mod+[        previous_window
map kitty_mod+f        move_window_forward
map kitty_mod+b        move_window_backward
map kitty_mod+`        move_window_to_top
map kitty_mod+1        first_window
map kitty_mod+2        second_window
map kitty_mod+3        third_window
map kitty_mod+4        fourth_window
map kitty_mod+5        fifth_window
map kitty_mod+6        sixth_window
map kitty_mod+7        seventh_window
map kitty_mod+8        eighth_window
map kitty_mod+9        ninth_window
map kitty_mod+0        tenth_window

# Tab management {{{
map kitty_mod+right    next_tab
map kitty_mod+left     previous_tab
map kitty_mod+t        new_tab
map kitty_mod+q        close_tab
map kitty_mod+l        next_layout
map kitty_mod+.        move_tab_forward
map kitty_mod+,        move_tab_backward
map kitty_mod+alt+t    set_tab_title
# You can also create shortcuts to go to specific tabs, with 1 being the first tab
# map ctrl+alt+1          goto_tab 1
# map ctrl+alt+2          goto_tab 2

# Layout management {{{
# You can create shortcuts to switch to specific layouts
# map ctrl+alt+1          goto_layout tall
# map ctrl+alt+2          goto_layout stack
# }}}

# Font sizes {{{
map kitty_mod+m increase_font_size
map kitty_mod+minus     decrease_font_size
map kitty_mod+backspace restore_font_size
# To setup shortcuts for specific font sizes, follow the example below:
# map kitty_mod+f6     set_font_size 10.0
# map kitty_mod+f7     set_font_size 20.5
# }}}
map kitty_mod+e         run_kitten text hints

map kitty_mod+p>f       run_kitten text hints --type path --program -

# Select a path/filename and open it with the default open program.
map kitty_mod+p>shift+f run_kitten text hints --type path

# Select a line of text and insert it into the terminal. Use for the
# output of things like: ls -1
map kitty_mod+p>l       run_kitten text hints --type line --program -

# Select words and insert into terminal.
map kitty_mod+p>w       run_kitten text hints --type word --program -

# The hints kitten has many more modes of operation that you can map to different
# shortcuts. For a full description run: kitty +kitten hints --help
# }}}

# Miscellaneous {{{
map kitty_mod+f11      toggle_fullscreen
map kitty_mod+u        input_unicode_character
map kitty_mod+f2       edit_config_file
  You can customize how the URLs are
# Open the kitty shell in a new window/tab/overlay/os_window to control kitty using commands.
map kitty_mod+escape   kitty_shell window

# You can have kitty remove all shortcut definition seen upto this point. Useful for
# instance, to remove the default shortcuts.
clear_all_shortcuts no
# }}}


# OS specific tweaks {{{

# Change the color of the kitty window's titlebar on macOS. A value of "system"
# means to use the default system color, a value of "background" means to use
# the default background color and finally you can use an arbitrary color, such
# as #12af59 or "red".
macos_titlebar_color system

# Hide the kitty window's title bar on macOS.
macos_hide_titlebar no
macos_option_as_alt yes

background_opacity 1.0
selection_foreground #000000
selection_background #FFFACD

background #1D2021
foreground #A89984
cursor #A89984
color0 #1D2021
color1 #FB543F
color2 #95C085
color3 #FAC03B
color4 #0D6678
color5 #8F4673
color6 #8BA59B
color7 #A89984
color8 #665C54
color9 #FB543F
color10 #95C085
color11 #FAC03B
color12 #0D6678
color13 #8F4673
color14 #8BA59B
color15 #FDF4C1```

**Comment by kovidgoyal** [comment]
Opening a new tab does not prompt for a name. You presumbaly used the shortcut to rename the existing tab instead.

**Comment by vulgrim** [comment]
It's kind of my fault. I used "ctrl + alt" as my mod key, so opening a new tab was  "mod + t" while renaming was "mod + alt + t". Then for opening a new tab I used "mod + t and for renaming "mod + r", but for some reason "mod+t" was still renaming the tab, like that "alt" was still stuck with renaming somehow (default setting for renaming are "mod + alt + t"). Maybe that is something worth checking out. 
Finally my current config is: 
``` 
kitty_mod ctrl+alt
map kitty_mod+r        new_tab
map kitty_mod+t        set_tab_title
```   

after this didn't work for some reason: 
``` 
kitty_mod ctrl+alt
map kitty_mod+t        new_tab
map kitty_mod+r        set_tab_title
```
